src/components/modules/tools.ts

```diff
@@ -472,3 +473,22 @@ export default class Tools extends Module {
     };
   }
 }
+
+/**
+ * What kind of plugins developers can create
+ */
+export enum toolTypes {
```

Comment:
Enums should be in PascalCase
```suggestion
export enum ToolTypes {
```

Reply:
Also, usually, enums go in singular form
```suggestion
export enum ToolType {
```